### PR TITLE
Additional editorial suggestions sections 2 and beyond

### PR DESCRIPTION
**Pull request by ianbjacobs**
Many fewer suggestions than the previous PR. :)

### CODE REVIEW

File: spec.bs [hunk 1]
```diff
@@ -376,22 +376,23 @@ To enroll a user for Secure Payment Confirmation, relying parties should call
 {{AuthenticationExtensionsClientInputs/payment}} [=WebAuthn Extension=]
 specified.
 
-Note: We currently require an extension in order to allow credential creation
-      in an iframe and to allow the browser to cache SPC credentials in the
-      absence of [=WebAuthn Conditional UI=]. Future versions of this
-      specification may remove the requirement for the extension.
+Note: We currently require an extension in order to allow cross-origin
+      credential creation in an iframe and to allow the browser to cache
```
Comment (stephenmcgruer) [inline spec.bs:380]:
```suggestion
Note: We currently require an extension in order to allow credential
      creation in a cross-origin iframe and to allow the browser to cache
```

File: spec.bs [hunk 2]
```diff
@@ -376,22 +376,23 @@ To enroll a user for Secure Payment Confirmation, relying parties should call
...
+      for the extension.
 
 # Authentication # {#sctn-authentication}
 
 To authenticate a payment via Secure Payment Confirmation, this specification
 defines a new [=payment method=], "[=secure-payment-confirmation=]". This
 payment method confirms the transaction with the user and then performs an
 [=authentication ceremony=] to authenticate the user and create a signed blob
-representing the transaction.
+representing the payment-specific information.
```
Comment (stephenmcgruer) [inline spec.bs:391]: Is this misleading? The signed blob is over the whole WebAuthn ceremony, not just the payment-specific bits. But *shrug*, don't particularly care here :D

File: spec.bs [hunk 3]
```diff
@@ -376,22 +376,23 @@ To enroll a user for Secure Payment Confirmation, relying parties should call
...
 
 At a high level, authentication for Secure Payment Confirmation is similar to
 [[webauthn-3]], with one major conceptual shift. Secure Payment Confirmation
-allows a third-party (e.g. the merchant) to trigger an authentication ceremony
+allows a third-party (e.g., the merchant) to trigger an authentication ceremony
```
Comment (stephenmcgruer) [inline spec.bs:395]: So TIL that American English requires a comma after `e.g.`... British English explicitly does not allow it. Fun!

**Review by stephenmcgruer — APPROVED** [review]

File: spec.bs (hunk 2 above)
Comment (ianbjacobs) [reply spec.bs:391]: I'll change it to the authentication ceremony.

**Pull request merged**